src/main/proto/vehicle/body/mirrors/v1/mirrors_service.proto

```diff
@@ -159,6 +159,18 @@ service BodyMirrors {
       returns (UpdateHeatedSideMirrorsSettingsResponse) {
     option (uprotocol.method_id) = 9;
   }
+
+  option (uprotocol.publish_topic) = {
+    id : 0x8000,
+    name : "side_mirrors",
+    message : "SideMirrors"
```

Comment:
previous version had 3 topics, I only see two. Also the topic messages are different